src/docs/build-deploy-and-maintain-apps/maintain-an-application.md

```diff
@@ -0,0 +1,498 @@
+
+**Database backups**
+
+The community supported [backup-container](https://github.com/bcgov/backup-container) is an image designed to support the automated backup of various types of database.  It is easy to set up and is well understood by our community of users, so you will be able to get support if needed.
+
+**PersistentVolume recovery**
+
+If you have a PVC using the 'netapp-file-backup' storage class, it could be recovered from backup, as that storage class is automatically backed up.  If you need to recover one of these backup volumes, **you will need the name of the PersistentVolume, not just the PVC name.**  You can get the volume name from the PVC details.  
+
+For example:
+```
+$ oc get pvc
+NAME           STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS           AGE
+backup-test    Bound    pvc-02e9d855-cd63-480d-a1d7-9b638b04f6ff   20Gi       RWX            netapp-file-backup     3d19h
+```
+In this case, you would store the volume name `pvc-02e9d855-cd63-480d-a1d7-9b638b04f6ff` in a safe place together with the PVC name and a description of its purpose.  The Platform Team will not be able to restore from backup without that volume name.
+
+**For extra protection, copy your DB backups to an S3 bucket**
+
+For more details, see the [Restoring Backup Volumes on OpenShift](https://docs.developer.gov.bc.ca/netapp-backup-restore/).
+
+**Images**
+
+Do your applications pull images directly from Docker Hub, ghcr.io, or another off-cluster source?  
+
+Can you be sure that the same version of the image will be available as long as you need it?  
+
+At times,  older images are removed from container repositories. This could happen because they are no longer supported or due to security vulnerabilities.  To ensure the constant availability of your images and safeguard against potential network issues that might hinder their retrieval, consider storing your images in Artifactory.  
+
+Reconsider the section:  [**Maintain images**](#image-maintenance) on this document for more information.
+
+### Periodic HA Testing
+After configuring HPAs and PDBs for your applications, test them periodically to make sure that they work the way that you expect and that your application remains available.  If you're testing in your Test or Dev environment, make sure they're configured the same as Prod so that you can test your Prod configuration.
+* Delete a pod in a Deployment or StatefulSet
+* In a DB replica set, delete a secondary member
+* Delete the primary member of a DB replica set
+    * Does the application or DB remain available?
+* Test your PDB by trying to delete more pods than allowed.  Does it work as expected?
+
+**Load testing**
+
+If you need to run a load test against your application, first check with the Platform Services team to ensure that the timing and scope of the test will not impact other users of the platform.
```

Comment:
Could also link to https://docs.developer.gov.bc.ca/prepare-to-load-test-application-on-openshift/

Reply:
Thanks, link added.